CHANGELOG.md

```diff
@@ -2,6 +2,105 @@
 
 This document outlines major changes between releases.
 
+## 0.106.0 "Zephyrantheses" (21 May 2024)
```

Comment:
Zephyranthes? The first one I couldn't find in `/usr/share/dict/words`.

Reply:
Fixed! Also, I'm open to suggestions, may be you have a better name.